pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -617,9 +669,8 @@ private PermissioningConfiguration permissioningConfiguration() {
     return permissioningConfiguration;
   }
 
-  private SynchronizerConfiguration buildSyncConfig(final SyncMode syncMode) {
-    checkNotNull(syncMode);
-    synchronizerConfigurationBuilder.syncMode(syncMode);
+  private SynchronizerConfiguration buildSyncConfig() {
+    synchronizerConfigurationBuilder.syncMode(DEFAULT_SYNC_MODE);
```

Comment:
It would be good not to remove the sync-mode flag.  I have an open PR that starts using it again. :)